File: Beginner/C/1008.c
#include <stdio.h>

int main(){

    int numero, horas;
    double salarioPorHora;

    scanf("%i", &numero);
    scanf("%i", &horas);
    scanf("%lf", &salarioPorHora);

    printf("NUMBER = %i\n", numero);
    printf("SALARY = U$ %.2lf\n", horas*salarioPorHora);

    return 0;
}

File: Beginner/JavaScript/problems/1008.js
export function problem(lines){
    let numero = parseInt(lines[0]);
    let horas = parseInt(lines[1]);
    let salarioPorHora = parseFloat(lines[2]);

    let salario = horas*salarioPorHora;
    console.log(`NUMBER = ${numero}`);
    console.log(`SALARY = U$ ${salario.toFixed(2)}`);
};